### PR TITLE
Updates from rspec-dev (2015-02-08)

### PR DESCRIPTION
**Pull request by myronmarston**
These are some updates, generated from rspec-dev's rake tasks.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -29,6 +29,6 @@ test_script:
 
 environment:
   matrix:
-    - ruby_version: '200'
```
Comment (bf4) [inline appveyor.yml]: is this change correct?

Comment (JonRowe) [inline appveyor.yml]: No?

Comment (bf4) [inline appveyor.yml]: Have a g'day. This central time zoner is going to be :zzz:

Comment (myronmarston) [inline appveyor.yml]: Originated in rspec/rspec-dev#113, which passed the build here and got merged but failed on rspec-core (see rspec/rspec-core#1861 and rspec/rspec-core#1868).  Not a big deal right now; once the rspec-core issue is dealt with we'll rebase the rspec-dev PR and push out updates to all the build scripts.  So ignore for now.

**Pull request closed without merging**